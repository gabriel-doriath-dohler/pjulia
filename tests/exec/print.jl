println("yo", "lo")
println("Hello, world!")
println("string ", 42)
println(true, false)
print(4, 2)
println(42)
print(true)
print(42)
print(false)
println()
println("Hello, world!")
println("Hello", ",", " ", "world", "!")
println(42, "newline\nafter newline\t", nothing)
println("\\\\")
println("\\\"")
