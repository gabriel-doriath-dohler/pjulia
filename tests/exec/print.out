yolo
Hello, world!
string 42
truefalse
4242
false
Hello, world!
Hello, world!
42newline
after newline	Nothing
\\
\"
